config.py

```diff
@@ -3,24 +3,28 @@
 TIMEOUT = 10
 
 PAUSE = 2
-EXTRA_LONG_PAUSE = 120
+EXTRA_LONG_PAUSE = 20
+
+EMAIL_SIGNUP = "rabotynet.test@gmail.com"
+FROM_SIGNUP = decode(b'cm9tYV9leHBlcnQ=').decode()
+
+EMAIL_FORGOT_PASSWORD = "rabotynet.test.fp@gmail.com"
```

Comment:
the same constant as `EMAIL_SIGNUP` and next constant is the same as `FROM_SIGNUP`, you may make only two constants, something like just `EMAIL` and `FORM_DECODE`